wagtail_localize/locales/tests.py

```diff
@@ -190,10 +277,14 @@ def post(self, post_data=None, locale=None):
         )
 
     def test_simple(self):
-        response = self.get()
+        # Test rendering the edit view with simple data
+        response = self.client.get(
```

Comment:
we now seem to have a mix of `self.client.get` like here, and `self.get` below (like in `test_invalid_language`). It would be nice to be consistent